2023-09-28 07:51:19 [http-nio-auto-2-exec-6] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=400 BAD_REQUEST, message=Order can not be closed if order is not ready., uri=/api/owners-tools/orders/close-order/654654-465465-555, timestamp=Thu, 28 Sep 2023 07:51:19 -0700}
2023-09-28 07:51:19 [http-nio-auto-2-exec-4] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=Order cannot be found and as such can not be closed. Please verify order id., uri=/api/owners-tools/orders/close-order/66, timestamp=Thu, 28 Sep 2023 07:51:19 -0700}
2023-09-28 07:51:19 [main] INFO  c.a.t.t.m.OrdersDbTests.EmptyTests -
                Starting EmptyTests using Java 17.0.6 with PID 387590 (started by brian in /media/brian/Backup 22/Documents/Tablet Backup/Documents/Courses/Java/personal projects/TonsOfTacosV1.2/back-end/tons-of-tacos)
2023-09-28 07:51:19 [main] DEBUG c.a.t.t.m.OrdersDbTests.EmptyTests -
                Running with Spring Boot v3.0.1, Spring v6.0.3
2023-09-28 07:51:19 [main] INFO  c.a.t.t.m.OrdersDbTests.EmptyTests -
                No active profile set, falling back to 1 default profile: "default"
2023-09-28 07:51:20 [main] DEBUG c.a.t.config.security.JwtAuthFilter -
                Filter 'jwtAuthFilter' configured for use
2023-09-28 07:51:20 [main] DEBUG c.a.t.c.s.FilterChainExceptionHandler -
                Filter 'filterChainExceptionHandler' configured for use
2023-09-28 07:51:20 [main] INFO  c.a.t.t.m.OrdersDbTests.EmptyTests -
                Started EmptyTests in 1.227 seconds (process running for 15.361)
2023-09-28 07:51:20 [main] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No orders found at all. Please contact your application team right away., uri=/api/owners-tools/orders/get-orders, timestamp=Thu, 28 Sep 2023 07:51:20 -0700}
2023-09-28 07:51:20 [main] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No customers found at all. Please contact your application team right away., uri=/api/owners-tools/customers/get-customers, timestamp=Thu, 28 Sep 2023 07:51:20 -0700}
2023-09-28 07:51:20 [http-nio-auto-2-exec-6] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No customer found with that UID. Please verify and try  again., uri=/api/owners-tools/customers/get-customer-uid/customerUid, timestamp=Thu, 28 Sep 2023 07:51:20 -0700}
2023-09-28 07:51:21 [http-nio-auto-2-exec-6] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=Order does not exist. Please verify order id and try again., uri=/api/owners-tools/orders/order-ready/88, timestamp=Thu, 28 Sep 2023 07:51:21 -0700}
2023-09-28 07:51:21 [http-nio-auto-1-exec-5] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=400 BAD_REQUEST, message=Customer name incorrectly formatted. Please consult the documentation., uri=/api/order/checkout, timestamp=Thu, 28 Sep 2023 07:51:21 -0700}
2023-09-28 07:51:21 [http-nio-auto-1-exec-7] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=403 FORBIDDEN, message=Bad credentials. username: m)Km7y{f0~nd$,hvNLOw0.F5FlP5u?5 password: 3x+e4:[li}4j$w.Ws03rro.q8ci|6+K5G9s5F56, uri=/api/owners-tools/login, timestamp=Thu, 28 Sep 2023 07:51:21 -0700}
2023-09-28 07:51:21 [http-nio-auto-1-exec-8] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=403 FORBIDDEN, message=Bad credentials. username: _T0w(S=rF5Yq]eQ|tg. password: ?aNwlfCd7glf(E&r)lLr}W?fT, uri=/api/owners-tools/login, timestamp=Thu, 28 Sep 2023 07:51:21 -0700}
2023-09-28 07:51:21 [main] INFO  c.a.t.t.TonsOfTacosApplicationTests -
                Starting TonsOfTacosApplicationTests using Java 17.0.6 with PID 387590 (started by brian in /media/brian/Backup 22/Documents/Tablet Backup/Documents/Courses/Java/personal projects/TonsOfTacosV1.2/back-end/tons-of-tacos)
2023-09-28 07:51:21 [main] DEBUG c.a.t.t.TonsOfTacosApplicationTests -
                Running with Spring Boot v3.0.1, Spring v6.0.3
2023-09-28 07:51:21 [main] INFO  c.a.t.t.TonsOfTacosApplicationTests -
                No active profile set, falling back to 1 default profile: "default"
2023-09-28 07:51:22 [main] INFO  c.a.t.t.TonsOfTacosApplicationTests -
                Started TonsOfTacosApplicationTests in 0.887 seconds (process running for 17.365)
2023-09-28 07:51:22 [http-nio-auto-2-exec-1] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No customer with that id found., uri=/api/owners-tools/customers/edit-customer-email/13/larry@larbo.com, timestamp=Thu, 28 Sep 2023 07:51:22 -0700}
2023-09-28 07:51:22 [http-nio-auto-2-exec-7] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=400 BAD_REQUEST, message=Email does not match formatting requirements, please consult the docs., uri=/api/owners-tools/customers/edit-customer-email/gd34-igjr/larrylarbo.com, timestamp=Thu, 28 Sep 2023 07:51:22 -0700}
2023-09-28 07:51:27 [main] INFO  c.a.t.t.s.o.o.GetAllOrdersTests$testThatDoesNotPolluteTheApplicationContextUris -
                Starting GetAllOrdersTests.testThatDoesNotPolluteTheApplicationContextUris using Java 17.0.6 with PID 387847 (started by brian in /media/brian/Backup 22/Documents/Tablet Backup/Documents/Courses/Java/personal projects/TonsOfTacosV1.2/back-end/tons-of-tacos)
2023-09-28 07:51:27 [main] DEBUG c.a.t.t.s.o.o.GetAllOrdersTests$testThatDoesNotPolluteTheApplicationContextUris -
                Running with Spring Boot v3.0.1, Spring v6.0.3
2023-09-28 07:51:27 [main] INFO  c.a.t.t.s.o.o.GetAllOrdersTests$testThatDoesNotPolluteTheApplicationContextUris -
                No active profile set, falling back to 1 default profile: "default"
2023-09-28 07:51:32 [main] DEBUG c.a.t.config.security.JwtAuthFilter -
                Filter 'jwtAuthFilter' configured for use
2023-09-28 07:51:32 [main] DEBUG c.a.t.c.s.FilterChainExceptionHandler -
                Filter 'filterChainExceptionHandler' configured for use
2023-09-28 07:51:33 [main] INFO  c.a.t.t.s.o.o.GetAllOrdersTests$testThatDoesNotPolluteTheApplicationContextUris -
                Started GetAllOrdersTests.testThatDoesNotPolluteTheApplicationContextUris in 5.994 seconds (process running for 7.345)
2023-09-28 07:51:34 [main] INFO  c.a.t.t.s.o.o.DeleteCustomerByUidTest$testThatDoesNotPolluteTheApplicationContextUris -
                Starting DeleteCustomerByUidTest.testThatDoesNotPolluteTheApplicationContextUris using Java 17.0.6 with PID 387847 (started by brian in /media/brian/Backup 22/Documents/Tablet Backup/Documents/Courses/Java/personal projects/TonsOfTacosV1.2/back-end/tons-of-tacos)
2023-09-28 07:51:34 [main] DEBUG c.a.t.t.s.o.o.DeleteCustomerByUidTest$testThatDoesNotPolluteTheApplicationContextUris -
                Running with Spring Boot v3.0.1, Spring v6.0.3
2023-09-28 07:51:34 [main] INFO  c.a.t.t.s.o.o.DeleteCustomerByUidTest$testThatDoesNotPolluteTheApplicationContextUris -
                No active profile set, falling back to 1 default profile: "default"
2023-09-28 07:51:35 [main] DEBUG c.a.t.config.security.JwtAuthFilter -
                Filter 'jwtAuthFilter' configured for use
2023-09-28 07:51:35 [main] DEBUG c.a.t.c.s.FilterChainExceptionHandler -
                Filter 'filterChainExceptionHandler' configured for use
2023-09-28 07:51:35 [main] INFO  c.a.t.t.s.o.o.DeleteCustomerByUidTest$testThatDoesNotPolluteTheApplicationContextUris -
                Started DeleteCustomerByUidTest.testThatDoesNotPolluteTheApplicationContextUris in 1.166 seconds (process running for 9.576)
2023-09-28 07:51:35 [http-nio-auto-2-exec-3] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No customer found with that UID. Please verify and try  again., uri=/api/owners-tools/customers/get-customer-uid/customerUid, timestamp=Thu, 28 Sep 2023 07:51:35 -0700}
2023-09-28 07:51:35 [http-nio-auto-2-exec-5] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No customer with that uid found. Please check formatting and verify customer exists., uri=/api/owners-tools/customers/delete-customer/85, timestamp=Thu, 28 Sep 2023 07:51:35 -0700}
2023-09-28 07:51:36 [http-nio-auto-2-exec-7] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=You have chosen a category that does not exist. Please check your spelling and formatting., uri=/api/menu/category, timestamp=Thu, 28 Sep 2023 07:51:36 -0700}
2023-09-28 07:51:36 [http-nio-auto-2-exec-3] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=No customer with that uid found. Please check formatting and verify customer exists., uri=/api/owners-tools/customers/edit-customer-phone/14/555.555.5156, timestamp=Thu, 28 Sep 2023 07:51:36 -0700}
2023-09-28 07:51:36 [http-nio-auto-2-exec-5] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=You have chosen a menu item that does not exist., uri=/api/menu/id, timestamp=Thu, 28 Sep 2023 07:51:36 -0700}
2023-09-28 07:51:36 [main] INFO  c.a.t.t.m.O.NonEmptyTests -
                Starting NonEmptyTests using Java 17.0.6 with PID 387847 (started by brian in /media/brian/Backup 22/Documents/Tablet Backup/Documents/Courses/Java/personal projects/TonsOfTacosV1.2/back-end/tons-of-tacos)
2023-09-28 07:51:36 [main] DEBUG c.a.t.t.m.O.NonEmptyTests -
                Running with Spring Boot v3.0.1, Spring v6.0.3
2023-09-28 07:51:36 [main] INFO  c.a.t.t.m.O.NonEmptyTests -
                No active profile set, falling back to 1 default profile: "default"
2023-09-28 07:51:37 [main] DEBUG c.a.t.config.security.JwtAuthFilter -
                Filter 'jwtAuthFilter' configured for use
2023-09-28 07:51:37 [main] DEBUG c.a.t.c.s.FilterChainExceptionHandler -
                Filter 'filterChainExceptionHandler' configured for use
2023-09-28 07:51:37 [main] INFO  c.a.t.t.m.O.NonEmptyTests -
                Started NonEmptyTests in 1.084 seconds (process running for 11.619)
2023-09-28 07:51:37 [http-nio-auto-2-exec-7] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=404 NOT_FOUND, message=Order item not updated. Verify order item is part of order., uri=/api/owners-tools/orders/update-order-item/654654-465465-555/88/2, timestamp=Thu, 28 Sep 2023 07:51:37 -0700}
2023-09-28 07:51:37 [http-nio-auto-2-exec-9] DEBUG c.a.t.e.GlobalExceptionHandler -
                {status code=400 BAD_REQUEST, message=We were unable to process your request. Please contact us when trying to order more than 10 of any given item., uri=/api/owners-tools/orders/update-order-item/654654-465465-555/3/12, timestamp=Thu, 28 Sep 2023 07:51:37 -0700}
